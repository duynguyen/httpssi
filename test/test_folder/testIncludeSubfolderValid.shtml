<div>
<!--#include file="sub_test_folder/included_sub.html" -->
</div>