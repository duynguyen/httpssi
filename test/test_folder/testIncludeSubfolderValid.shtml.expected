<div>
Included in sub folder.
</div>